src/main/java/net/masterthought/cucumber/FeatureScenario.java

```diff
@@ -0,0 +1,46 @@
+package net.masterthought.cucumber;
+
+/**
+ * Created by suci on 11/17/17.
```

Comment:
should be removed as this can be taken from git history

Reply:
done removing creator